package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "mdn-data",
+  "version": "1.0.0",
+  "description": "Open Web data by the Mozilla Developer Network",
+  "repository": {
```

Comment:
i would say add `main` entrypoint field, which will point to `index.js`, which in turn will expose all json files from `./css/` folder.